ipwb/replay.py

```diff
@@ -691,15 +691,19 @@ def handler(signum, frame):
     # respWithLinkHeader = getLinkHeaderAbbreviatedTimeMap(path, datetime)
     # resp.headers['Link'] = respWithLinkHeader.replace('\n', ' ')
 
-    if status[0] == '3':
+    if status[0] == '3' and isUri(resp.headers.get('Location')):
```

Comment:
That's the point of this check. We do not want to rewrite `Location` if it is not a full URI. SW should be able to handle that condition.

Reply:
> SW should be able to handle that condition

Does Reconstruct handle this condition with this line excluded? I have yet to test it. Have you?

Reply:
No, I have not tested it, but it should work. Even if it doesn't, the functionality is better than before where were incorrectly rewriting.

Reply:
By the way, I used `resp.headers.get('Location')` instead of `resp.headers['Location']` in the `if` condition to avoid any exceptions if for whatever reason the `Location` header was not present in the captures record although it had `3xx` status.